fpfilter.pl

```diff
@@ -183,7 +184,8 @@
     if(length($ref) > 1 || length($var) > 1) {
```

Comment:
Why not fix it while we're looking at it - I think the only change needed is to  line 186:

```
<    
     if(length($ref) == length($var)) {

>         if(length($ref) == length($var) || (length($ref) > 1 && length($var) > 1) ) {


```